backend/apps/events/views.py

```
@@ -1,15 +1,201 @@
+        data=json.dumps(payload),
+        content_type="application/json",
+    )
+    reservation_request.user = request.user
+    reservation_request.residence = residence
+
+    response = SpaceReservationCreateView().post(reservation_request, space_id=space_id)
+    if response.status_code != 201:
+        try:
+            data = json.loads(response.content.decode("utf-8"))
+            detail = data.get("detail") or "No se pudo crear la reserva del evento interno."
+        except Exception:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)